File: java/impl/src/test/java/org/taymyr/lagom/javadsl/openapi/generate/empty/EmptyService.java
package org.taymyr.lagom.javadsl.openapi.generate.empty;

import com.lightbend.lagom.javadsl.api.Descriptor;
import com.lightbend.lagom.javadsl.api.Service;
import org.taymyr.lagom.javadsl.openapi.OpenAPIService;
import org.taymyr.lagom.javadsl.openapi.OpenAPIUtils;

public interface EmptyService extends OpenAPIService  {
    @Override
    default Descriptor descriptor() {
        return OpenAPIUtils.withOpenAPI(Service.named("test"));
    }
}

File: scala/impl/src/test/scala/org/taymyr/lagom/scaladsl/openapi/generate/empty/EmptyService.scala
package org.taymyr.lagom.scaladsl.openapi.generate.empty

import com.lightbend.lagom.scaladsl.api.Descriptor
import com.lightbend.lagom.scaladsl.api.Service
import com.typesafe.config.Config
import org.taymyr.lagom.scaladsl.openapi.OpenAPIService
import org.taymyr.lagom.scaladsl.openapi.OpenAPIServiceImpl

trait EmptyService extends OpenAPIService with Service {
  override def descriptor: Descriptor = Service.named("test").withOpenAPI()
}

class EmptyServiceImpl(override val config: Config) extends EmptyService with OpenAPIServiceImpl {}
